drafts/HTTP-Response-Compression.adoc

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Setting HTTP Response Compression with Open Liberty"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/28316667
+author_github: https://github.com/mrsaldana
+seo-title: Setting HTTP Response Compression with Open Liberty
+seo-description: You can now specify in the server configuration which HTTP endpoints will consider compressing HTTP responses. HTTP response compression may be used to reduce bandwidth between HTTP clients and the server.
+blog_description: You can now specify in the server configuration which HTTP endpoints will consider compressing HTTP responses. HTTP response compression may be used to reduce bandwidth between HTTP clients and the server.
+---
+= Setting HTTP Response Compression with Open Liberty
+Manuel Saldana <https://github.com/mrsaldana>
+:imagesdir: /
+:url-prefix:
+:bl: pass:[ +]
+:star: pass:[*]
+:add: pass:[+]
+
+
+It is now possible to configure Open Liberty so that HTTP responses are considered for compression. By compressing the sizes of HTTP responses, bandwidth is reduced, which in turn decreases exchanges times between HTTP clients and Open Liberty servers. Sounds good so far... but, what does the statement `_HTTP responses are considered for compression_` actually mean?
```

Comment:
'decreases exchanges times` to `decreases exchange times`

Reply:
changed, thanks!